.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,15 +8,11 @@ about: Create a bug report to help us improve
 _Progress of the issue based on the [Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_
 
 - [ ] 1. The issue provides a reproduction available on [Github](https://github.com/dotansimha/graphql-code-generator-issue-sandbox-template), [Stackblitz](https://stackblitz.com/github/dotansimha/graphql-code-generator-issue-sandbox-template) or [CodeSandbox](https://codesandbox.io/s/github/dotansimha/graphql-code-generator-issue-sandbox-template)
-
-_Make sure to fork this template and run `yarn generate` in the terminal._
-
-_Please make sure the Codegen and plugins version under `package.json` matches yours._
-
+> Make sure to fork this template and run `yarn generate` in the terminal.
+> 
+> Please make sure the Codegen and plugins version under `package.json` matches yours.
```

Comment:
`graphql-eslint` does not depend on codegen and its plugins, the reproduction template should be different